chart/tests/pod-template-file_test.yaml

```diff
@@ -22,14 +22,17 @@ tests:
     asserts:
       - isKind:
           of: Pod
-      - equal:
+      - isNotNull:
           path: spec.containers[0].image
-          value: dummy_image
```

Comment:
I could find a way to assert that this value should be the default `defaultAirflowRepository` without hardcoding the test with `apache/airflow:1.10.12` (copying from values.yaml). If that hardcoded value in the assertion is fine, I can change the test back to `equal` assert.